### PR TITLE
Add active filter

### PR DESCRIPTION
**Pull request by GMadorell**
[sc-10737]

### CODE REVIEW

File: com/coralogix/xdr/v1/filter/security_test_filter.proto [hunk 1]
```diff
@@ -55,13 +56,15 @@ message LogFilterCounts {
...
   map<string, CustomFilterValueCounts> custom_filters = 7;
+  map<string, google.protobuf.UInt32Value> active = 8;
 }
 
 message SecurityRuleFilterCounts {
   // This is defined on security rule level
   map<string, google.protobuf.UInt32Value> severity = 1;
   map<string, google.protobuf.UInt32Value> policy = 2;
   ProviderFilterCounts provider = 3;
+  map<string, google.protobuf.UInt32Value> active = 4;
```
Comment (stsatlantis) [inline com/coralogix/xdr/v1/filter/security_test_filter.proto:67]: Why adding to both places ( LogFilterCounts and RuleFilterCounts) when the request is related to the log only

Comment (GMadorell) [reply com/coralogix/xdr/v1/filter/security_test_filter.proto:67]: Yeah, that's right, deleting it!

**Review by stsatlantis — APPROVED** [review]

**Review by xBATx — APPROVED** [review]

**Comment by xBATx** [comment]
@GMadorell shouldn't we add these active filter also to the `resource_filter`?

**Comment by stsatlantis** [comment]
> @GMadorell shouldn't we add these active filter also to the `resource_filter`?

It'll be a different task :)

**Pull request merged**